sapi/cli/php_cli.c

```diff
@@ -1208,6 +1212,11 @@ int main(int argc, char *argv[])
 	 */
 	argv = save_ps_args(argc, argv);
 
+#ifdef PHP_WIN32
```

Comment:
> Re setmode() - I meant in the block of code that calls setmode, just to keep the PHP_WIN32 startup stuff together.

They have different and unrelated purposes, I don't see why we have the necessity to keep them together.